pkg/manager/manager_tasks.go

```diff
@@ -492,3 +495,46 @@ func (s *singleton) neededGenerate(scenes []*models.Scene, sprites, previews, ma
 	}
 	return &totals
 }
+
+func excludeFiles(files []string, patterns []string) ([]string, int) {
+	if patterns == nil {
+		logger.Infof("No excludes in config.")
+		return files, 0
+	} else {
+		var results []string
+		var exclCount int
+		var fileRegexps []*regexp.Regexp
+
+		for _, pattern := range patterns {
+			reg, err := regexp.Compile(strings.ToLower(pattern))
+			if err != nil {
+				logger.Errorf("Aborting Exclude:%v", err)
+				return files, 0
+			}
```

Comment:
As a user, if I made a mistake with one of the regex patterns, I would still expect the scan to exclude using the patterns that were valid. I think not applying any exclude patterns due to a single invalid pattern is unintuitive behaviour.

Reply:
Yeah , it was my first step after the must compile change ...
I'll print an error warning and let it continue